docs/API.md

```diff
@@ -0,0 +1,26 @@
+# API
+
+## Socket events
+
+### Client
+
+Client-side API sends socket events to the server.
+
+sendToServer.js handles events dispatching.
+
+The serverObject provides methods to use :
+
+- `serverObject.sendNewUser()` returns a client ID (String)
+- `serverObject.sendDeleteUser()` returns a string telling that a client has disconnected.
```

Comment:
what are all the possible values of this string? What is the semantics behind each string?